Source/Flow/Private/FlowAsset.cpp

```diff
@@ -71,16 +76,45 @@ void UFlowAsset::PostDuplicate(bool bDuplicateForPIE)
 	}
 }
 
+void UFlowAsset::PostLoad()
+{
+	Super::PostLoad();
+
+	// If we removed or moved a flow node blueprint (and there is no redirector) we might loose the reference to it resulting
+	// in null pointers in the Nodes FGUID->UFlowNode* Map. So here we iterate over all the Nodes and remove all pairs that
+	// are nulled out.
+	
+	TSet<FGuid> NodesToRemoveGUID;
+
+	for (auto& [Guid, Node] : GetNodes())
+	{
+		if (!IsValid(Node))
+		{
+			NodesToRemoveGUID.Emplace(Guid);
+		}
+	}
+
+	for (const FGuid& Guid : NodesToRemoveGUID)
+	{
+		UnregisterNode(Guid);
+	}
+}
+
 EDataValidationResult UFlowAsset::ValidateAsset(FFlowMessageLog& MessageLog)
 {
 	// validate nodes
 	for (const TPair<FGuid, UFlowNode*>& Node : Nodes)
 	{
-		if (Node.Value)
+		if (IsValid(Node.Value))
 		{
-			if (!IsNodeClassAllowed(Node.Value->GetClass()))
+			FText FailureReason;
+			if (!IsNodeClassAllowed(Node.Value->GetClass(), &FailureReason))
```

Comment:
Validate will now report inaccessible flow nodes being used in a flow graph (that is, they're in an inaccessible plugin).